src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
 // write code here
+const tree = document.querySelector('.tree');
+
+tree.addEventListener('click', (e) => {
+  const title = e.target.closest('li');
+  const childs = [...title.children];
```

Comment:
The variable `childs` is used to store the children of the `title` element. However, the term 'childs' is not standard English; consider renaming it to `children` for better readability.